lib.nix

```diff
@@ -1,49 +1,41 @@
-{ nodeImplementation }:
+{ nodeImplementation ? "jormungandr" }:
 
 let
-  # iohk-nix can be overridden for debugging purposes by setting
-  # NIX_PATH=iohk_nix=/path/to/iohk-nix
-  iohkNix = import (
-    let try = builtins.tryEval <iohk_nix>;
-    in if try.success
-    then builtins.trace "using host <iohk_nix>" try.value
-    else
-      let
-        spec = builtins.fromJSON (builtins.readFile ./iohk-nix.json);
-      in builtins.fetchTarball {
-        url = "${spec.url}/archive/${spec.rev}.tar.gz";
-        inherit (spec) sha256;
-      }) {};
+  sources = import ./nix/sources.nix;
+  iohkNix = import sources.iohk-nix { application = "daedalus"; };
+  # TODO: can we use the filter in iohk-nix instead?
```

Comment:
cant put this into `iohk-nix` because this is a very strict whitelist, based on what daedalus itself needs in the source
other projects need other files